src/components/ProgressBar.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import Translator from '../constants/strings';
+
 
 function ProgressBar(props) {
     return (
         <div className="progress">
             <div className="progress__step">
-                {'Step ' + (props.currentPage + 1) + '/8'}
+                {Translator.trans('ProgressBarStep') + ' ' + (props.currentPage + 1) + '/8'}
```

Comment:
theres a case for placeholders:
```js
Translator.trans('ProgressBarStep', {'%current_step%': props.currentPage + 1, '%total_steps%': NUM_TOTAL_STEPS});
```
where is `ProgressBarStep` looks like `step %current_step% of %total_steps%`.
Placeholders useful if in various languages some parts can be placed in different places.